### PR TITLE
[TLV-2021] First Commit to Get Started

### PR DESCRIPTION
**Pull request by shar1z**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,28 +1,201 @@
-Copyright [2016-2019] [devopsdays]
```
Comment (phrawzty) [inline LICENSE:1]: Please do not overwrite `/LICENSE`

File: README.md [hunk 2]
```diff
@@ -1,34 +1,190 @@
-[![Build Status](https://circleci.com/gh/devopsdays/devopsdays-web.svg?branch=main)](https://circleci.com/gh/devopsdays/devopsdays-web) [![Build status](https://ci.appveyor.com/api/projects/status/3lobrrssphdb20xd?svg=true)](https://ci.appveyor.com/project/DevOpsDays/devopsdays-web)
```
Comment (phrawzty) [inline README.md:1]: Please do not overwrite `/README`

File: content/events/2021-tel-aviv/welcome.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</div> -->
+
+<div class = "row">
+  <div class = "col-md-2">
+    <strong>Sponsors</strong>
+  </div>
+  <div class = "col-md-8">
+    {{< event_link page="sponsor" text="Sponsor the conference!" >}}
+  </div>
+</div>
```
Comment (phrawzty) [inline content/events/2021-tel-aviv/welcome.md:73]: You may wish to comment this out until you've got non-boilerplate content to display. Your call.

File: data/events/2021-tel-aviv.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  - name: "John Doe"
+  - name: "Jane Smith"
+    twitter: "devopsdays"
+  - name: "Sally Fields"
+    employer: "Acme Anvil Co."
+    github: "devopsdays"
+    facebook: "https://www.facebook.com/sally.fields"
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "https://mattstratton.com"
+    image: "sally-fields.jpg"
```
Comment (phrawzty) [inline data/events/2021-tel-aviv.yml:61]: Please populate this with the actual humans in the local team.

File: data/events/2021-tel-aviv.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+organizer_email: "tel-aviv@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: samplesponsorname
+    level: gold
```
Comment (phrawzty) [inline data/events/2021-tel-aviv.yml:69]: This has no visual effect but you probably don't want it.

**Review by phrawzty — CHANGES_REQUESTED** [review]
_Also_, please don't add the `hugo` binary (can't comment on binaries). :)

File: data/events/2021-tel-aviv.yml (hunk 5 above)
Comment (mattstratton) [reply data/events/2021-tel-aviv.yml:69]: fwiw it's okay to leave this in as a helpful format reference, but as @phrawzty you might want to delete it if that doesn't help you :)

Comment (shar1z) [reply data/events/2021-tel-aviv.yml:69]: Thank you both - I'll make the edits.  I just wanted to have a placeholder, and only after I committed did I see the John & Jane Doe.  Hopefully in the coming days, I'll actually have the final dates.  So maybe I'll hold.  

Really appreciate it.

Comment (phrawzty) [reply data/events/2021-tel-aviv.yml:69]: @shar1z  imho at this point you probably just want to close this PR and open a new one when you're ready.

**Review by shar1z — COMMENTED** [review]
@mattstratton @phrawzty - Ok - I think I'm ready to announce the event - and I also think I made the relevant edits from the previous commits.

LMK if we're good to merge.

Thanks!

**Comment by mattstratton** [comment]
Please see @phrawzty's comments on the files to exclude (the hugo binary and LICENSE) and remove them from this PR; as dan mentioned, it might be easier to just start a new PR at this point?

File: LICENSE (hunk 1 above)
Comment (mattstratton) [reply LICENSE:1]: @shar1z this commit needs to be removed; you can't change the LICENSE :)

File: README.md (hunk 2 above)
Comment (mattstratton) [reply README.md:1]: @shar1z this is another file that should not be changed in the PR

**Comment by mattstratton** [comment]
@shar1z I was able to remove the two files (README and LICENSE) from this PR, but the `hugo` binary I cannot remove from the PR. We cannot merge this when the binary is attached; I would suggest rebasing your fork/branch and submitting a new PR?

File: LICENSE (hunk 1 above)
Comment (mattstratton) [reply LICENSE]: I took care of this part

File: README.md (hunk 2 above)
Comment (mattstratton) [reply README.md]: I think I resolved this

**Pull request merged**